src/org/aigents/nlp/lg/Dictionary.java

```diff
@@ -0,0 +1,59 @@
+package org.aigents.nlp.lg;
+
+import java.util.*;
```

Comment:
It is highly recommended to refer to only the packages and specific classes that you actually use.